woof-code/support/cros_image.sh

```diff
@@ -0,0 +1,74 @@
+wget -O- https://github.com/dimkr/devsus/releases/latest/download/devsus-templates.tar.gz | tar -xzf-
```

Comment:
TODO: produce the partition layout + ext4 file system here, instead of downloading a blank image